tests/integration/test_install_basic.py

```diff
@@ -416,6 +416,7 @@ def test_rewrite_outline_table(pipenv_instance_private_pypi):
         assert 'colorama = "*"' in contents
 
 
+@pytest.mark.skip(reason="Plette is more strict then pipfile ...")
```

Comment:
Let's just axe this test case -- I believe I did in my other PR around categories.